client/post-editor/editor-sharing/accordion.jsx

```diff
@@ -82,18 +83,23 @@ const EditorSharingAccordion = React.createClass( {
 	},
 
 	hideSharing: function() {
-		return this.props.site &&
-			this.props.site.jetpack &&
-			! this.props.site.isModuleActive( 'publicize' ) &&
-			! this.props.site.isModuleActive( 'sharedaddy' ) &&
-			! this.props.site.isModuleActive( 'likes' );
+		return ( this.props.site &&
+				this.props.site.jetpack &&
+				! this.props.site.isModuleActive( 'publicize' ) &&
+				! this.props.site.isModuleActive( 'sharedaddy' ) &&
+				! this.props.site.isModuleActive( 'likes' ) ) ||
+			( this.props.site &&
+				this.props.site.jetpack &&
+				this.props.postType === 'page' &&
+				! this.props.site.isModuleActive( 'sharedaddy' ) &&
+				! this.props.site.isModuleActive( 'likes' ) );
```

Comment:
I think we can make this method a bit cleaner, and we should probably be using the `postTypeSupports` selector to avoid hardcoding:

```js
const { site, postTypeSupportsPublicize } = this.props;

if ( ! site || ! site.jetpack ) {
    return false;
}

let sharingModules;
if ( postTypeSupportsPublicize ) {
    sharingModules = [ 'publicize', 'sharedaddy', 'likes' ];
} else {
    sharingModules = [ 'sharedaddy', 'likes' ];
}
return sharingModules.every( ( module ) => ! site.isModuleActive( module ) );
```